src/common/checksum.c

```diff
@@ -210,6 +239,9 @@ daos_csummer_init(struct daos_csummer **obj, struct csum_ft *ft,
 	result->dcs_algo = ft;
 	result->dcs_chunk_size = chunk_bytes;
 	result->dcs_srv_verify = srv_verify;
+	result->dcs_dedup = dedup;
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?